### PR TITLE
Enabling GA for regression as well as travis

### PR DESCRIPTION
**Pull request by info-rchitect**

### CODE REVIEW

File: .github/workflows/regression_tests.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    - name: Gem Install Origen 
+      run: gem install origen 
+    - name: Setup Dependencies
+      run: origen -v
+    - name: Set Origen to debug mode
+      run: origen m debug
+    - name: Run Spec tests
+      run: origen app_gen:test --regression
+    - name: Verify Building website
+      run: origen web compile --no-serve
```
Comment (priyavadan) [inline .github/workflows/regression_tests.yml]: dont think app generators has web compile

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-  - gem install bundler -v '< 2'
-script:
-  - bundle exec origen -v
-    #- bundle exec origen specs
-  - bundle exec origen app_gen:test --regression
-    #- bundle exec origen web compile --no-serve
-  - bundle exec origen lint --no-correct
-env:
-  - ORIGEN_GEM_USE_FROM_SYSTEM=false ORIGEN_GEM_MANAGE_BUNDLER=false
+# language: ruby
```
Comment (priyavadan) [inline .travis.yml:1]: I would not comment these out. I probably have to change this in my PR too. We can have both travis and GA enabled simultaneously for a bit

**Review by priyavadan — CHANGES_REQUESTED** [review]

File: .github/workflows/regression_tests.yml (hunk 1 above)
Comment (info-rchitect) [reply .github/workflows/regression_tests.yml]: Yep fixed in latest commit

**Review by priyavadan — APPROVED** [review]

**Comment by priyavadan** [comment]
lets wait on the CI to finish before merging, should pass IMO

**Comment by info-rchitect** [comment]
Merging as per @priyavadan

**Pull request merged**